### DISCUSSION TITLE
Can you update swizzle components?

### DISCUSSION THREAD

**Discussion by jcubic** [facebook/docusaurus#10781] [Q&A] [answered]
I was just wondering, there are new versions of Docusaurus and I have old version of swizzle components. Is there a way to swizzle again and based on diff add again my changes? This way I can update swizzle components. Or maybe there is a better way to update them?

It would be nice if it produced a conflict when updating modified files so it's easy to merge user changes with new versions from Docusaurus.

**Comment by slorber** [comment, marked as answer]
We don't have any other way than swizzling them again.
If you use Git you can then see the diff from what was previously committed, and find a way to merge changes from Docusaurus with your former version.

We'd like to improve things but it's probably not so easy. Here are related issues:
- https://github.com/facebook/docusaurus/issues/5734
- https://github.com/facebook/docusaurus/issues/6114

There's also shadcn/ui CLI that somehow has a similar distribution model and similar problems to solve. They have a cli `diff` command to help. Maybe we could reuse or take inspiration from it. See also https://github.com/shadcn-ui/ui/discussions/790
